apps/web/components/user/recent-activity.tsx

```
@@ -31,6 +32,38 @@ type Activity = {
 export function RecentActivity({ userId }: { userId: string }) {
   const trpc = useTRPC();
   const { data: activities } = useQuery(trpc.profile.getRecentActivities.queryOptions({ userId }));
+  const [showAll, setShowAll] = useState(false);
+  const [isSmallScreen, setIsSmallScreen] = useState(true);
+  const SMALL_SCREEN_BREAKPOINT = 640;
+
+  const throttle = (func: Function, delay: number) => {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)